src/lib/constants.ts

```diff
@@ -40,6 +40,7 @@ export const USD = 'USD'
 export const RBTC = 'rBTC'
 export const STRIF = 'stRIF'
 export const USDRIF = 'USDRIF'
+export const tRIF = 'tRIF'
```

Comment:
Is this necessary? Isn't there other places that shows `tRIF` correctly without adding a constant for it? Addionally we agreed in another [PR](https://github.com/RootstockCollective/dao-frontend/pull/1285#discussion_r2256655115) we should keep constants uppercase, it would need to change to `TRIF`.